### ISSUE TITLE
On landing page can we increase the size of the logo and make the color contrast of words stronger (perhaps black?)

### ISSUE THREAD

**Issue by eefronson** [BlueCodeSystems/nacmis#334]
I am attaching a screenshot of what it currently looks like and a screenshot of what the NAC website currently looks like. We should aim to get it as close as possible to the current one for consistency. 
<img width="1227" alt="screen shot 2018-07-25 at 8 57 12 am" src="https://user-images.githubusercontent.com/38062748/43184490-0e32f0f2-8fe9-11e8-87f7-3afd79e36bb4.png">
<img width="1176" alt="screen shot 2018-07-25 at 8 57 52 am" src="https://user-images.githubusercontent.com/38062748/43184491-0e66ea4c-8fe9-11e8-9815-1bd46a59b4ee.png">

**Comment by tobiasmakai** [comment]
Try 
![logo_reviewed](https://user-images.githubusercontent.com/25362528/43202364-9b882bd4-901b-11e8-95b3-e01fcf0d455b.png)
